### PR TITLE
[artifactory-ha] - Add custom labels in Nginx deployment and Artifactory statefulset's

### LINKED ISSUE #505: [artifactory-ha] - Add custom labels in Nginx deployment and Artifactory statefulset's
**Is this a BUG REPORT or FEATURE REQUEST?**: FEATURE REQUEST

**Which chart**: artifactory-ha

**What you expected to happen**: To have the chance to define custom labels for the nginx deployment and primary & node sateefulset k8s objects

### PR DESCRIPTION
**Pull request by guillemtanya**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:  To have the chance to define custom labels for the nginx deployment and primary & node sateefulset k8s objects

**Which issue this PR fixes**: fixes #505

@amithins
@danielezer 
@eldada 
@rimusz

### CODE REVIEW

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -780,6 +781,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.primary.javaOpts.jmx.ssl`              | Enable SSL           |  `false` |
 | `artifactory.primary.javaOpts.other`            | Artifactory primary node additional java options |                     |
 | `artifactory.primary.persistence.existingClaim` | Whether to use an existing pvc for the primary node | `false`            |
+| `artifactory.node.labels`                       | Artifactory primary node labels                  | `{}`                |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md]: This should say artifactory member nodes labels

File: stable/artifactory-ha/templates/nginx-deployment.yaml [hunk 2]
```diff
@@ -11,6 +11,9 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
     component: {{ .Values.nginx.name }}
+{{- if .Values.artifactory.primary.labels }}
```
Comment (danielezer) [inline stable/artifactory-ha/templates/nginx-deployment.yaml]: The if should check for nginx.labels

**Comment by danielezer** [comment]
Please add this feature to the artifactory chart as well

File: stable/artifactory-ha/README.md (hunk 1 above)
Comment (guillemtanya) [reply stable/artifactory-ha/README.md]: Resolved

File: stable/artifactory-ha/templates/nginx-deployment.yaml (hunk 2 above)
Comment (guillemtanya) [reply stable/artifactory-ha/templates/nginx-deployment.yaml]: Resolved

**Comment by guillemtanya** [comment]
Added into "artifactory" chart

**Pull request merged**